packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -232,15 +232,19 @@ export const getOrderFilledEvents = async (
       const outcomeTokenOne = await Promise.all(
         events
           .filter((event) => {
-            return market.clobTokenIds[0] == event?.args?.takerAssetId.toString();
+            return [event?.args?.takerAssetId.toString(), event?.args?.makerAssetId.toString()].includes(
```

Comment:
This filter was wrong, the way it worked before was filtering out sells and keeping buys.